src/util.cpp

```diff
@@ -65,8 +73,17 @@ ::mstch::node wrapYAMLNode(const YAML::Node &node, ScalarConversionFn fn)
             {
                 const std::string name = it->first.as<std::string>();
                 const YAML::Node &value = it->second;
-                context[name] = wrapYAMLNode(value);
+                context[name]
+                    = wrapYAMLNodeImpl(value, nullptr, markLast, false);
+            }
+
+            if (markLast && isLast)
+            {
+                // The key "last" should be synced with the {{key}} element of
+                // chimera::mstch::ClangWrapper
+                context["last"] = true;
             }
+
```

Comment:
Wouldn't this add "{last: true}" to every map element in the parent sequence whenever the map element isn't empty?

Current:
```
classes
  - name: ClassA
  - name: ClassB
    last: true  # because this map element is the last one in the parent sequence
```

Suggested:
```
classes
  - name: ClassA
    last: true  # because this map element is not empty
  - name: ClassB
    last: true  # because this map element is not empty
```

Reply:
Hmm, I think it would only apply to the elements _inside_ the map (i.e. fields inside the map).  But again, I am not sure this is useful for anything, so we can remove it.

Reply:
I see. I didn't get your intention correctly.

How about we revisit this if we find a use case for this feature?

Reply:
Yeah, we don't need this feature now.